broadcaster/_backends/memory.py

```diff
@@ -7,10 +7,10 @@
 class MemoryBackend(BroadcastBackend):
     def __init__(self, url: str):
         self._subscribed: typing.Set = set()
-        self._published: asyncio.Queue = asyncio.Queue()
+        self._published: typing.Optional[asyncio.Queue] = None
 
     async def connect(self) -> None:
-        pass
```

Comment:
Sonuvabitch this is such a simple fix I'm mad at myself for not realizing it was creating the queue before the starlette event loop was available.

Good catch!

Reply:
The only reason I caught it was I created the same bug for myself with the redis rewrite, then when I looked at the in memory version, I knew straight-away what the problem was